### PR TITLE
email fix - members

### PR DESCRIPTION
**Pull request by Kiran-Alex**
Now When the user is adding members it gives a response if the email format is invalid

### CODE REVIEW

File: components/invitation/InviteMember.tsx [hunk 1]
```diff
@@ -28,9 +28,9 @@ const InviteMember = ({
       role: availableRoles[0].id,
     },
     validationSchema: Yup.object().shape({
-      email: Yup.string().email().required(),
+      email: Yup.string().email().required('Email is required'), // Added error message
```
Comment (devkiran) [inline components/invitation/InviteMember.tsx:31]:
```suggestion
      email: Yup.string().email().required('Email is required'),
```

File: components/invitation/InviteMember.tsx [hunk 2]
```diff
@@ -28,9 +28,9 @@ const InviteMember = ({
       role: availableRoles[0].id,
```
Comment (devkiran) [inline components/invitation/InviteMember.tsx:28]: We have added prettier format to standardise formatting across the source files. Please merge main and then run `npm run format`.

File: components/invitation/InviteMember.tsx [hunk 3]
```diff
@@ -98,6 +99,9 @@ const InviteMember = ({
                 ))}
               </select>
             </div>
+            {formik.touched.email && formik.errors.email && (
+              <div className="text-red-600">{formik.errors.email}</div>
+            )}
```
Comment (devkiran) [inline components/invitation/InviteMember.tsx:104]: Can you please align the error message properly on the UI?

**Review by devkiran — CHANGES_REQUESTED** [review]

**Comment by Kiran-Alex** [comment]
The Error Is Now Shown as This, Please Review This and I'm actively looking for the feedback
image

**Comment by devkiran** [comment]
@Kiran-Alex Can you please check the error state on other input? Here is how error state looks on login page.

Login-BoxyHQ

**Comment by Kiran-Alex** [comment]
The UI and error State is Updated, Similar to the Signup

image

**Review by devkiran — CHANGES_REQUESTED** [review]
- Can you run `npm run format` to format the code?
- Let's remove the `onBlur` property and handle the validation after the form submission.

Other than that, it looks good to me. We'll merge it once you make the changes.

**Comment by Kiran-Alex** [comment]
email fix #3 - Now the error validation is done when the user clicks on the button instead of any place on the modal.

**Review by devkiran — APPROVED** [review]

**Pull request merged**